datafusion/src/lib.rs

```diff
@@ -182,6 +182,35 @@
 //! * declare and use user-defined aggregate functions ([`AggregateUDF`](physical_plan::udaf::AggregateUDF))
 //!
 //! you can find examples of each of them in examples section.
+//!
+//! ## Examples
+//!
+//! Examples are located in [datafusion-examples directory](https://github.com/apache/arrow-datafusion/tree/master/datafusion-examples)
+//!
+//! Here's how to run them
+//!
+//! ```bash
+//! git clone https://github.com/apache/arrow-datafusion
+//! cd arrow-datafusion
+//! # Download test data
+//! git submodule update --init
+//! export PARQUET_TEST_DATA=parquet-testing/data
```

Comment:
This stuff is unfortunate - it would be nice to have the examples be able to find this on their own (as we have done with arrow-rs).

I have filed a ticket here: https://github.com/apache/arrow-datafusion/issues/467

Reply:
@alamb Great idea :+1: 
I'll try to work on #467 then

Thank you  :slightly_smiling_face: